### ISSUE TITLE
Lots of numbers being spit out

### ISSUE THREAD

**Issue by MikeyBeez** [ideasman42/nerd-dictation#10]
Thank you for writing this interesting project.  It's running, but it's spitting out a lot of garbage along with the text.  

❯ ./nerd-dictation begin
0.09997663497924805
0.09870014190673829
0.09955344200134278
0.09974346160888672
0.09971175193786622
0.0929502010345459
0.09946784973144532
0.09947595596313477
0.0925527572631836
0.09944138526916504
0.09245476722717286
0.09949836730957032
0.09236202239990235
0.09945592880249024
0.09939346313476563
0.0923090934753418
0.09901008605957032
THIS0.09907612800598145
 IS0.039521551132202154
0.09932670593261719
0.09929046630859376
 ANOTHER0.07741460800170899
0.09929213523864747
0.09936389923095704
 TERRORIST0.015120840072631841
0.09926352500915528
ST0.09925565719604493
0.0896986484527588
0.09934697151184083
0.09947404861450196
0.09257588386535645
0.09938035011291504
0.09136066436767579
0.09934458732604981
0.06850967407226563
0.09943637847900391
0.09936747550964356
0.09154710769653321
0.09944114685058594
0.09195122718811036
0.09947142601013184

How do I suppress all these logits?

**Comment by MikeyBeez** [comment]
Never mind.  Duh!  I figured it out.  I don't run this in a terminal.  I run it as a server, and I use it in a terminal, or anywhere else for that matter.  Thank you.  This is really cool!  I may try to add punctuation; so I can say period.  Of course, I think I can just press a period.  I need to play with this.  It's very very cool, however.  I'm actually using it here.   I'm entering the note  by dictating,  not by typing.   And  yeah,  I'm getting used to adding the punctuation myself and doing the initial cap word myself.   I'm such a shitty typist,  this should be a really big help for me.   I'm sorry that this note is so long.   It's just that this dictation is really fun.

**Comment by ideasman42** [comment]
Hey I'm glad you liked  this, I'm also dictating right now.

I've never seen the kind of output you're referencing, it seems somehow a debug mode has been enabled which isn't default (or could be new I suppose if you're using a different version of VOSK).

Did you manage to suppress the output or are you just ignoring it?

**Comment by MikeyBeez** [comment]
I made a video that shows it.  https://www.youtube.com/watch?v=msC7KNzqClU

**Comment by ideasman42** [comment]
Thanks for posting, there were indeed prints, now fixed.

I don't run this directly from a terminal often so I didn't notice they were left in.

Thanks for posting the video on this too :)

**Comment by MikeyBeez** [comment]
I posted it in my Deep Learning group too.  https://www.facebook.com/groups/DeepNetGroup
